packages/mdc-radio/_radio-theme.scss

```diff
@@ -135,7 +136,7 @@ $density-config: (
   $query: feature-targeting.all()
 ) {
   $feat-structure: feature-targeting.create-target($query, structure);
-  $offset: ($ripple-size - $size) / 2;
+  $offset: math.div($ripple-size - $size, 2);
```

Comment:
You'll have to rebase after https://github.com/material-components/material-components-web/pull/7157 is merged.